### PR TITLE
Revert one of the previous changes where an image was changed to stable from nightly build

### PR DESCRIPTION
**Pull request by vishnuchalla**

### CODE REVIEW

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml [hunk 1]
```diff
@@ -49,7 +44,6 @@ tests:
     - chain: ipi-gcp-post
     pre:
     - chain: ipi-gcp-pre
-    - chain: create-infra-move-ingress-monitoring-registry
```
Comment (paigerube14) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13.yaml:52]: I think we want to move ingress/monitoring pods to infra nodes here. Was there a reason you removed them?

**Pull request merged**